pkg/rbac/cache.go

```diff
@@ -13,11 +15,16 @@ type Cache struct {
 	tokenReviews        map[string]*tokenReviewResult
 	tokenReviewsPending map[string][]chan *tokenReviewResult
 	tokenReviewsLock    sync.Mutex
+	shared              sharedList
+	sharedLock          sync.Mutex
```

Comment:
This sharedLock can be moved to the `sharedList` struct because we don't need to lock at this level.

Reply:
Ah, right!